### DISCUSSION TITLE
hook example

### DISCUSSION THREAD

**Discussion by abernier** [ComfortablyCoding/strapi-plugin-transformer#78] [Q&A]
Hi,

I was looking for an example on how to use ˋhooks.postResponseTransform(ctx)` method, would you have one?

thank you

**Comment by ComfortablyCoding** [comment]
Sure. 

Scenario:
You want to return an additional top level property `time` that returns the current time in the response.

To do so using a hook would look like so:
```js
module.exports = ({ env }) => ({
  // ..
 'transformer': {
    enabled: true,
    config: {
     // ..
      hooks: {
        postResponseTransform : (ctx) => {
          ctx.body.time = time();
        }
      },
    // ..
    }
  },
  // ..
});
```

**Reply by abernier** [reply]
ok this is great, where can we have documentation about `ctx` structure?

**Reply by ComfortablyCoding** [reply]
For the most part it is a [koa context object](https://koajs.com/#context)
